src/dialects/db2/query-generator-typescript.ts

```diff
@@ -33,4 +36,26 @@ export class Db2QueryGeneratorTypeScript extends AbstractQueryGenerator {
       'ORDER BY NAME;',
     ]);
   }
+
+  removeIndexQuery(tableName: TableNameOrModel, indexNameOrAttributes: string | string[], options: RemoveIndexQueryOptions) {
+    if (options) {
+      rejectInvalidOptions(
+        'removeIndexQuery',
+        this.dialect.name,
+        REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS,
+        new Set(),
```

Comment:
In the other functions we would call this `REMOVE_INDEX_QUERY_SUPPORTED_OPTIONS`. That might still be nice to have here, but maybe we should define it just before `rejectInvalidOptions` instead of at the start of the file (which we do for the other instances). @ephys do you have an opinion on this?

Reply:
I've added this inside the if statement, LMK if this is the desired location